frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.styled.tsx

```diff
@@ -17,12 +17,16 @@ const SettingsAsideBlock = styled.div`
   padding: 2rem;
   border-right: 1px solid ${color("border")};
   background-color: ${color("white")};
+
+  height: 45.125rem;
```

Comment:
Where does this value come from? It's pretty random to me - even if we convert it to pixel values we get `722px` which seems like a really random number

Reply:
nit: I kind of get the same question. Like do `45rem` or `46rem` still work? Because those numbers look a lot better. I'd prefer 45rem since it also rounds to 720px for aesthetic.